pages/docs/end-user-guide/nominating.mdx

```diff
@@ -0,0 +1,129 @@
+to significant changes on the Avail Mainnet. However does provides valuable insights for making an informed decision. Key parameters to consider include the threshold and minimum nominated values:
+
+- **Threshold**: This represents the minimum AVL token value required for staking and is predetermined as part of the AVL network settings. It can be modified through a governance process. Atbthis moment on Avail Testnet this value is 0. However this is subject to change in Mainnet.
+
+- **Minimum Nominated**: This value is dynamic, recalculated every era based on the total staked percentage. As more stake enters the network, this value increases. To earn rewards, stakers must ensure their stake surpasses this minimum nominated threshold.
+
+If your stake amount is significantly larger than the threshold and minimum nominated amounts you can stake directly. If your stake is smaller, slightly above these values and you do not want to keep checking your stake it is then recommended to join a Nomination Pool
+
+<img src="/img/nominating/target-values-01.png" width="100%" height="100%" />
+
+
+### How to Nominate
+
+#### 1. Navigate to Staking Accounts Page
+Before beginning the nominating process, ensure you have a wallet containing AVL and have backed up the mnemonic phrase. As mentioned earlier, nominating involves staking or delegating on Avail.
+Start the nomination process by navigating to the [<ins>Avail Staking Accounts Page</ins>](https://goldberg.avail.tools/#/staking/actions). Click on the Nominator `+Nominator` button.
+
+<img src="/img/nominating/nominate-01.png" width="100%" height="100%" />
+
+
+#### 2. Select Wallet & Stake Amount
+You will need select the wallet you wish to nominate from, also the amount. On the amount always leave some AVL for fees, so only nominate 95%-98% of your tokens. Levaing 2%-5% for fees on nominating or any other tranactions you may want to perform.
+Also select where you would like your rewards to acrue to. You have the following options
+
+- **Stash Account (Increase the amount at stake):** This refers to a scenario where the user increases the amount of AVL (Avail token) at stake with their rewards in their stash account. This is like auto-compunding, your rewards are added to your stake and will increase your overall stake for the next era.
+
+- **Stash Account (Do not increase the amount at stake):** In contrast to the previous point, here users choose not to increase the amount of AVL at stake in their stash account. Yur rewards will be sent to this account but ot staked.
+
+- **Controller Account:** The controller account is separate from the stash account and is used to manage the nomination process. This is legacy and will be discontinued going forward.
```

Comment:
@Genesis3800 this probably needs double checking from our side.

Reply:
@robin-rrt pls clarify. I don't understand what exactly needs double checking

Reply:
Controller is legacy and we should remove from docs or stipulate. You will see in new UI for goldberg we do not have option to use controller accounts.

Read here to understand that they will be depecated
https://support.polkadot.network/support/solutions/articles/65000182218#:~:text=Controller%20accounts%20are%20being%20deprecated,account%20using%20the%20Staking%20Dashboard.

Reply:
Typo

Your rewards will be sent to this account but not staked.